### ISSUE TITLE
Improve Register form language

### ISSUE THREAD

**Issue by sutch** [Drupal4Gov/Drupal-GovCon-2015#67]
URL: /ticket/register
- [ ] Rename second section from Free Ticket to Free Ticket Recipient.
- [ ] Add language to top of first section, something like: "Provide information about yourself, the person submitting this registration."
- [ ] Add language to top of second section, something like: "Provide information about the ticket recipient."

**Comment by timwood** [comment]
It's possible to register for a ticket for someone else. So the first "free ticket" section is not necessarily for the person submitting the registration. Try logging in and registering for another ticket, for someone else.

**Comment by sutch** [comment]
@timwood Yes, and this change is about clearly identifying each section to make it clear that the first section is for the person performing the registration and that the second section (and third, fourth, etc) are for the person receiving a ticket.
